pytd/pandas_td/__init__.py

```diff
@@ -178,6 +178,7 @@ def read_td_query(
     """
     if params is None:
         params = {}
+    params["force_tdclient"] = True
```

Comment:
It would be better to add note or comment on docstring pytd enforces to use td-client for pandas_td.

Reply:
Noted 30d4aa2b05d0ae012c37d627f9add851a997436a